test/integration/entry-integration.js

```diff
@@ -395,4 +401,20 @@ describe('Entry Api', () => {
         })
     })
   })
+
+  describe('read plainClientApi', () => {
+    /**
+     * @type {import('../../lib/contentful-management').PlainClientAPI}
+     */
+    let plainClient
+    before(async () => {
+      plainClient = await getDefaultSpace(true)
```

Comment:
```suggestion
       plainClient = initPlainClient({ spaceId: TestDefaults.spaceId })
```

so we don't need that change in `helpers.js`